testData/solutions/win32.json5

```diff
@@ -7556,6 +7563,70 @@
                 "inverseCapabilitiesTransformations": {
                     "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.MouseSonar": "MouseSonarConfig.value"
                 }
+            },
+            "configureDoubleClickWidth": {
+                "type": "gpii.windows.nativeSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "functionName": "DoubleClickWidth"
+                },
+                "supportedSettings": {
+                    "DoubleClickWidthConfig": {
+                        "schema": {
+                            "title": "Double-click rectangle width",
+                            "description": "Changes the width of the rectangle in which a second click should fall for OS to consider the mouse action a double-click.",
+                            "type": "integer",
+                            "default": 4,
+                            "min": 1,
+                            "max": 32,
+                            "divisibleBy": 1
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "DoubleClickWidthConfig": {
+                            "transform": {
+                            "type": "fluid.transforms.value",
+                            "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.DoubleClickWidth",
+                            "outputPath": "value"
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.DoubleClickWidth": "DoubleClickWidthConfig.value"
+                }
+            },
+            "configureDoubleClickHeight": {
+                "type": "gpii.windows.nativeSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "functionName": "DoubleClickHeight"
+                },
+                "supportedSettings": {
+                    "DoubleClickHeightConfig": {
+                        "schema": {
+                            "title": "Double click time",
+                            "description": "Changes the width of the rectangle in which a second click should fall for OS to consider the mouse action a double-click.",
+                            "type": "integer",
+                            "default": 4,
```

Comment:
It would be great if you can describe in a JIRA (and link it from here) the fact that we still need to investigate what is the real value range (if any), and why you have decided to add such value range.

Reply:
Doing it.

Reply:
Can you make a reference to https://issues.gpii.net/browse/GPII-3732 inside a comment?

Reply:
Sure, on top of the solution schema?

Reply:
I don't know, something like this:

```diff
@@ -7572,6 +7572,7 @@
                 },
                 "supportedSettings": {
                     "DoubleClickWidthConfig": {
+                        // Couldn't find a real value range for this setting, see https://issues.gpii.net/browse/GPII-3732
                         "schema": {
                             "title": "Double-click rectangle width",
                             "description": "Changes the width of the rectangle in which a second click should fall for OS to consider the mouse action a double-click.",
```

Reply:
Great, following with that.

Reply:
This is done.